firebase-firestore/ktx/ktx.gradle

```diff
@@ -66,4 +87,19 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     testImplementation "com.google.truth:truth:$googleTruthVersion"
     testImplementation 'androidx.test:core:1.2.0'
+    //TODO: remove this after decoder's pr approved to use Truth to replace assertFailWith
+    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion"
+
+    androidTestImplementation project(path: ':firebase-firestore:ktx')
```

Comment:
Is this dep because of the issue with Android Studio?

Reply:
I have removed this dependency.